api/v1alpha/subschema/artifacts.schema.json

```diff
@@ -40,6 +40,13 @@
             "directory"
           ],
           "additionalProperties": false
+        },
+        "retain": {
+          "description": "Compress folders into zip files, which can then be downloaded as artifacts.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
```

Comment:
That'd make this feature supposedly (according to the spec) available for Sauce Orchestrate too, but it's not.

Reply:
hmm... this is unexpected.

Reply:
Added separate artifacts schema for SO.